CHANGELOG.md

```diff
@@ -240,7 +240,7 @@ The following changes have been implemented but not released yet:
 
 ### Bugfixes
 
-- `getWellKnownSolid` fetches well known solid from the server's root by default.
+- `getWellKnownSolid` fetches well-known solid from the server's root by default.
```

Comment:
That was already the default behavior, wasn't it? The change is that it will no longer search for it at the Pod root.